### PR TITLE
feat(access): add flapping detect config page

### PR DESCRIPTION
**Pull request by ysfscream**
https://emqx.atlassian.net/browse/ED-716

### CODE REVIEW

**Comment by ysfscream** [comment]
Hi, @Meggielqk Could you help us quickly review the document about fault jitter detection if you have time?

**Review by Kinplemelon — APPROVED** [review]

File: src/i18n/General.js [hunk 1]
```diff
@@ -195,4 +195,36 @@ export default {
...
+    zh: '进行连接抖动检测时的时间窗口长度',
+    en: 'The length of time window for flapping detection',
+  },
+  maxCount: {
+    zh: '最大断连次数',
+    en: 'Max Disconnection Count',
+  },
+  maxCountDesc: {
+    zh: '在一个检测时间窗口内，MQTT 连接客户端被允许的最大断连次数',
+    en: 'Maximum disconnects allowed for a MQTT client within a detection window time',
```
Comment (Meggielqk) [inline src/i18n/General.js:220]: Maximum allowed number of disconnections for an MQTT client within a specified detection time window

File: src/i18n/General.js [hunk 2]
```diff
@@ -195,4 +195,36 @@ export default {
...
+    zh: '启用抖动检测',
+    en: 'Enable Flapping Detection',
+  },
+  enableDesc: {
+    zh: '启用连接抖动的检测功能',
+    en: 'Enable connection flapping detection feature',
+  },
+  windowTime: {
+    zh: '检测时间窗口',
+    en: 'Detection Window Time',
```
Comment (Meggielqk) [inline src/i18n/General.js:208]: Detection Time Window

File: src/i18n/General.js [hunk 3]
```diff
@@ -195,4 +195,36 @@ export default {
...
+    zh: '最大断连次数',
+    en: 'Max Disconnection Count',
+  },
+  maxCountDesc: {
+    zh: '在一个检测时间窗口内，MQTT 连接客户端被允许的最大断连次数',
+    en: 'Maximum disconnects allowed for a MQTT client within a detection window time',
+  },
+  banTime: {
+    zh: '封禁时长',
+    en: 'Ban Time',
```
Comment (Meggielqk) [inline src/i18n/General.js:224]: Ban Duration

File: src/i18n/General.js (hunk 1 above)
Comment (ysfscream) [reply src/i18n/General.js:220]: Thanks. Fixed on: https://github.com/emqx/emqx-dashboard5/pull/1572

File: src/i18n/General.js (hunk 3 above)
Comment (ysfscream) [reply src/i18n/General.js:224]: Thanks. Fixed on: https://github.com/emqx/emqx-dashboard5/pull/1572

File: src/i18n/General.js (hunk 2 above)
Comment (ysfscream) [reply src/i18n/General.js:208]: Thanks. Fixed on: https://github.com/emqx/emqx-dashboard5/pull/1572

**Pull request merged**